src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsStartup.cs

```diff
@@ -28,11 +29,9 @@ public void ConfigureServices(IServiceCollection services)
             options.MaxFormMappingRecursionDepth = 5;
             options.MaxFormMappingCollectionSize = 100;
         })
+            .AddWebAssemblyComponents()
             .AddServerComponents()
-            .AddWebAssemblyComponents(options =>
-            {
-                options.PathPrefix = "/WasmMinimal";
-            });
+            .AddHubOptions(ho => { });
```

Comment:
Was this necessary? If so, why?

Reply:
I don't think so, but @javiercn can correct me if I'm wrong